config/routes.rb

```diff
@@ -1,5 +1,14 @@
 Rails.application.routes.draw do
-  devise_for :users, controllers: { registrations: "registrations"}
+
+  devise_for :users, :skip => [:passwords], controllers: {
+    registrations: "registrations",
+    sessions: "sessions"}
+
   root "home#index"
+
+
+  resources :trucks, only: [:index]
   resources :trucks, only: [:index, :show]
```

Comment:
It looks like you have two lines here that both include `trucks#index` on the same level- can you get rid of line 10 if you're keeping line 11 please?